packages/replayer/src/replayer.ts

```diff
@@ -170,7 +174,7 @@ export const replayEvents: ReplayEventsFn = async (options) => {
     logLevel,
     sessionData,
     moveBeforeClick: true,
-    acceleratePlayback: accelerate,
+    virtualTime: accelerate ? { enabled: false } : { enabled: true },
```

Comment:
Unless I am going crazy:

```suggestion
    virtualTime: accelerate ? { enabled: true } : { enabled: false },
```

Reply:
No you're right! I don't know how I missed this.